src/TypeScript.UnitTests/EslintBridgeClient/EslintBridgeClientTests.cs

```diff
@@ -34,35 +34,64 @@ namespace SonarLint.VisualStudio.TypeScript.UnitTests.EslintBridgeClient
     [TestClass]
     public class EslintBridgeClientTests
     {
+        [TestMethod]
+        public void MefCtor_CheckIsExported()
+        {
+            MefTestHelpers.CheckTypeCanBeImported<TypeScript.EslintBridgeClient.EslintBridgeClient, IEslintBridgeClient>(null, new[]
+            {
+                MefTestHelpers.CreateExport<IEslintBridgeHttpWrapper>(Mock.Of<IEslintBridgeHttpWrapper>()),
+                MefTestHelpers.CreateExport<ILogger>(Mock.Of<ILogger>())
+            });
+        }
+
+        [TestMethod]
+        public async Task InitLinter_HttpWrapperCalledWithCorrectArguments()
+        {
+            var analysisConfiguration = new Mock<IAnalysisConfiguration>();
+            analysisConfiguration.Setup(x => x.GetEnvironments()).Returns(new[] {"env1", "env2"});
+            analysisConfiguration.Setup(x => x.GetGlobals()).Returns(new[] {"global1", "global2"});
+
+            var rules = new[] { new Rule { Key = "test1" }, new Rule { Key = "test2" } };
+
+            var httpWrapper = SetupHttpWrapper("init-linter", assertReceivedRequest: receivedRequest =>
+            {
+                var initLinterRequest = receivedRequest as InitLinterRequest;
+                initLinterRequest.Should().NotBeNull();
+                initLinterRequest.Rules.Should().BeEquivalentTo(rules);
+                initLinterRequest.Environments.Should().BeEquivalentTo("env1", "env2");
+                initLinterRequest.Globals.Should().BeEquivalentTo("global1", "global2");
+            });
+
+            var testSubject = CreateTestSubject(httpWrapper.Object, analysisConfiguration.Object);
+            await testSubject.InitLinter(rules);
+
+            httpWrapper.Verify(x => x.PostAsync("init-linter", It.IsAny<object>()), Times.Once);
+            httpWrapper.VerifyNoOtherCalls();
+        }
+
         [TestMethod]
         public async Task AnalyzeJs_HttpWrapperCalledWithCorrectArguments()
         {
-            var httpWrapper = new Mock<IEslintBridgeHttpWrapper>();
-            httpWrapper
-                .Setup(x => x.PostAsync("analyze-js", It.IsAny<object>()))
-                .ReturnsAsync((string)null);
+            var httpWrapper = SetupHttpWrapper("analyze-js", assertReceivedRequest: receivedRequest =>
+            {
```

Comment:
I'm using a helper method to do the assertion instead of `It.Is` -- It.Is was cumbersome and when it fails, it just says that there was no invocation with X params and it doesn't tell you what went wrong. This is friendlier for debugging.